### DISCUSSION TITLE
Customized Azure IPAM deployment monthly cost estimate

### DISCUSSION THREAD

**Discussion by serottez** [Azure/ipam#326] [Q&A] [answered]
Hi, I am wondering what the monthly costs of IPAM would be assuming a deployment using the following customizations:
- A private Azure Container Registry (ACR)
- Deploying the app engine to an Azure Function using an Azure storage account blob container to store function metadata.

Thank you in advance!
(This question was also asked in another [discussion](https://github.com/Azure/ipam/discussions/82#discussioncomment-10698582))

**Comment by DCMattyG** [comment, marked as answer]
Hi @serottez, I've worked up an Azure Price Calculator estimate here. I can only quote based on PAY-AS-YOU-GO pricing, so your costs will likely vary depending on the agreements & discounting you have with Microsoft.

https://azure.com/e/7d3700d42a3047879d466825e5339c34

Hope that helps!

**Reply by DCMattyG** [reply]
Please keep in mind, there isn't any special pricing per-se for Azure IPAM, the costs are just the sum of each of the components deployed to support the solution.
